Sources/Views/ClipsViews.swift

```diff
@@ -10,358 +10,149 @@ import PointFreePrelude
-          ],
-          "Point-Free clips"
-        ),
-        .p(
-          attributes: [],
-          .text(
-            """
-            A collection of some of our favorite moments from Point-Free episodes.
-            """)
-        )
-      )
-    ),
-    .ul(
-      attributes: [
-        .class([
-          Class.margin([.mobile: [.all: 0]]),
-          Class.padding([.mobile: [.all: 0], .desktop: [.leftRight: 2, .topBottom: 0]]),
-          Class.type.list.styleNone,
-          Class.flex.wrap,
-          Class.flex.flex,
-        ]),
-        .style(maxWidth(.px(1080)) <> margin(topBottom: nil, leftRight: .auto)),
-      ],
-      .fragment(
-        clips.enumerated().map { index, clip in
-          cardView(
-            title: clip.title,
-            subtitle: nil,
-            blurb: clip.blurb,
-            url: siteRouter.path(for: .clips(.clip(videoID: clip.vimeoVideoID))),
-            imageURL: clip.posterURL,
-            index: index
-          )
+    CenterColumn {
+      div {
+        div {
+          div {
+            iframe()
+              .attribute("src", "https://player.vimeo.com/video/\(clip.vimeoVideoID)?pip=1")
+              .attribute("frameborder", "0")
+              .attribute("allow", "autoplay; fullscreen")
+              .attribute("allowfullscreen")
+              .inlineStyle("width", "100%")
+              .inlineStyle("height", "100%")
+              .inlineStyle("position", "absolute")
+            script()
+              .attribute("src", "https://player.vimeo.com/api/player.js")
+          }
+          .inlineStyle("width", "100%")
+          .inlineStyle("position", "relative")
+          .inlineStyle("padding-bottom", "56.25%")
         }
-      )
-    ),
-  ]
+        .inlineStyle("box-shadow", "0rem 1rem 20px rgba(0,0,0,0.2)")
+        .inlineStyle("margin", "0 4rem", media: .desktop)
+      }
+    }
+    .inlineStyle("margin-top", "-4rem")
+    .inlineStyle("padding-bottom", "4rem")
+  }
 }
 
-private func videoView(
-  videoID: VimeoVideo.ID
-) -> Node {
-  return .div(
-    attributes: [
-      .class([outerVideoContainerClass]),
-      .style(outerVideoContainerStyle),
-    ],
-    .iframe(
-      attributes: [
-        .class([innerVideoContainerClass]),
-        .src("https://player.vimeo.com/video/\(videoID)?pip=1"),
-        Attribute("frameborder", "0"),
-        Attribute("allow", "autoplay; fullscreen"),
-        Attribute("allowfullscreen", ""),
-      ]
-    ),
-    .script(attributes: [.async(true), .src("https://player.vimeo.com/api/player.js")])
-  )
+struct CenterColumn<Content: HTML>: HTML {
+  @HTMLBuilder let content: Content
+  public var body: some HTML {
+    tag("pf-center-column") {
+      content
+        .inlineStyle("max-width", "1280px")
+        .inlineStyle("width", "100%")
+        .inlineStyle("margin", "0 auto")
+    }
+    .inlineStyle("width", "100%")
+    .inlineStyle("box-sizing", "border-box")
+    .inlineStyle("display", "block")
+  }
```

Comment:
Experimenting with a view component that does the minimal work to center some content in the screen with auto margins and max width 1280px. I think we could replace all instances of our hard coded 1280px with this.